config/config.go

```diff
@@ -0,0 +1,40 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/davecgh/go-spew/spew"
+
+	"github.com/spf13/viper"
+)
```

Comment:
By convention, imported packages in a file should be separated into three separated blocks:

1st block: Golang library packages
2nd block: Your project's own dependencies (local packages)
3rd block: 3rd party packages

```suggestion
import (
	"fmt"
	"log"
	"os"

	"github.com/davecgh/go-spew/spew"
	"github.com/spf13/viper"
)
```